### DISCUSSION TITLE
Multiple Eufy Home Base 2

### DISCUSSION THREAD

**Discussion by TompaNation** [fuatakgun/eufy_security#217] [Q&A]
Hi

I'm just curios if anyone have tried this together with 2 home bases 2?
I'm just able to get the read on one of them.

Best regards
Tomas Dunér

**Comment by fuatakgun** [comment]
Did not understand the problem?

**Reply by TompaNation** [reply]
Hi

I have 2 Home Bases 2 in my home network. One Home Base in the house and one in the garage. 
And they are available in the Eufy security:
![image](https://user-images.githubusercontent.com/61341892/149653449-b0c02f4c-bef3-4da6-9aa3-cde3595dce0d.png)

But when I look in the Eufy Security in the configuration in HA:

![image](https://user-images.githubusercontent.com/61341892/149653666-b54895ba-827f-42c8-9fbb-81d555af3f2a.png)

There is only 1 Home Base available. 
![image](https://user-images.githubusercontent.com/61341892/149653754-8df3ffac-3486-40ae-b755-b2f47133f029.png)

I have tried to restart everything.

Any ideas if it is possible to have multiple Home bases?

Best regads
Tomas

**Reply by fuatakgun** [reply]
Hey, i have tested multiple times with multiple users, multiple bases should be working.

If you are volunteering, share devices (one camera for each base) with me and i can debug

**Reply by TompaNation** [reply]
Hi

What do you need? Right now I do not have a cam on one of the Home Bases, I only have a door magnet sensor. And for now this is the only thing I like to have.
But I guess this should not be any problems. 

Best regards
Tomas

**Reply by fuatakgun** [reply]
Ok, before sharing the devices, can you please enable debug logging following readme and share all logs after restarting home assistant about `eufy_security`

You can reduct anything you want if you are gonna share it over here. If you don't want to bother about reduction of fields, drop me an email over fuatakgun@gmail.com

**Reply by TompaNation** [reply]
Just a VERY stupied question, how do I start debug logging? 
Do I need to put something like this in my YAMAL file?

logger:
  default: info
  logs:
    XXXXSomething.EufyXXXXX: debug

**Comment by fuatakgun** [comment]
please follow readme, captured over there
